tests/sources/xml/test_oai_dc.py

```diff
@@ -1,12 +1,11 @@
 import pytest
-
 from bs4 import BeautifulSoup
 
 import transmogrifier.models as timdex
 from transmogrifier.sources.xml.oaidc import OaiDc
 
 
-def create_oai_dc_source_record_stub(xml_insert: str = "") -> BeautifulSoup:
+def create_oaidc_source_record_stub(xml_insert: str = "") -> BeautifulSoup:
```

Comment:
I like the default, I'll update my with that

Reply:
+1

Reply:
I had agreed with the comment until the strikethrough, lolz!  

See [this comment](https://github.com/MITLibraries/transmogrifier/pull/177/files#r1611753661) about how maybe keeping the function closer to the tests is handy, given we can modify the signature to set values as needed in the stub.  Maybe good fodder for our discussion today.

To me, feels like either approach is still kind of on the table.